### ISSUE TITLE
post-install-hook?

### ISSUE THREAD

**Issue by djay** [collective/collective.themesitesetup#1]
It would be useful to call a script/view/themefragment sometimes after a profile is installed. Could help with install steps that aren't covered by GS. For example installing ambidexterity config.

**Comment by datakurre** [comment]
Would you have an example? Also, I don't know what kind of config there is with ambidexterity, but we should take into account that themesitesetup can only execute restricted python.

**Comment by djay** [comment]
ambidexterity has a syncrhonisation button that alters the dexterity and FTI definitions based on its config in portal_resources. so a url to perform that action could help solve that install. But probably making ambidexterity have GS support is better.

But in general I think it would be useful to have a series of urls that could be called via a subrequest that either point to themefragments (that can only install what can be done via restricted python but that does include creating content for example). 
or could call some existing FS based view to perform an extra action. Perhaps add a special user which GS can't do?

**Comment by datakurre** [comment]
> But in general I think it would be useful to have a series of urls that could be called via a subrequest that either point to themefragments (that can only install what can be done via restricted python but that does include creating content for example).

That makes sense and would cover both calling theme fragments and filesystem views.

Alternatives would be having new section in manifest.cfg with some ini-syntax for configuring URLs to call, or implementing a new GS step in themesitesetup that would read URLs and their configuration from GS XML.

Should it be paths instead of URLs (/ would mean the root of the current Plone site)?

**Comment by djay** [comment]
Yes I was just thinking paths in the manifest and some way to handle
relative paths so they are relative to the theme.
